src/content/docs/en/guides/backend/xata.mdx

```diff
@@ -6,9 +6,142 @@ service: Xata
 stub: true
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
 
-[Xata](https://xata.io) is a **Serverless Data Platform** that combines the features of a relational database, a search engine, and an analytics engine by exposing a single consistent REST API.
+
+[Xata](https://xata.io) is a **Serverless Data Platform** because it combines the features of a relational database, a search engine, and an analytics engine by exposing a single consistent REST API.
+
+## Adding a database with Xata
+
+### Prerequisites
+
+- A [Xata](https://app.xata.io/signin) account with a created database (you can use the sample database from the Web UI).
+- An Access Token (`XATA_TOKEN_API`).
+
+### Environment configuration
+
+The quickest way to integrate is via the Xata CLI
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install -g @xata.io/cli@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add -g @xata.io/cli@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn global install @xata.io/cli@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+The package will expose the `xata` command to your `$PATH` and now it's possible to initialize the integration:
+
+```shell
+xata init
+```
+
+:::note
+You can just execute the binary with `npx`, `pnpm dlx`, or `yarn dlx` if preferred.
+:::
+
+Once you follow the prompt, you should have a `.env` file in the root of your project with the following variables defined:
+
+```ini title=".env"
+XATA_API_KEY=hash_key
+
+# Xata branch that will be used
+# if there's not a xata branch with
+# the same name as your git branch
+XATA_FALLBACK_BRANCH=main
+```
+
+To have IntelliSense for your environment variables, edit or create the file `env.d.ts` in your `/src` directory to make your environment variables type-safe:
```

Comment:
This seems to me to be the important part of environment config, for an Astro project specifically? Maybe this is the only thing in this section (assuming it's easy enough to include setting up your environment as a prerequisite?)

Note: I might be wrong! You might need this entire section in order to integrate with Astro. If so, do let me know! I just know it's pretty common for people to PR guides that include setting up their *own* parts of the 3rd party, which is usually explained and doc'ed elsewhere. So, I need to check!  :smile:

Reply:
No no, absolutely right.

I'll just leave the `env.d.ts` part as a reassurance of how to use the env variables better with typescript in an Astro project, if that's alright. Wdyt?